src/Web/Internal/FFI.js

```diff
@@ -1,13 +1,6 @@
 "use strict";
 
 exports._unsafeReadProtoTagged = function (nothing, just, name, value) {
-  if (typeof window !== "undefined") {
-    var ty = window[name];
-    if (ty != null && value instanceof ty) {
-      return just(value);
-    }
-    return nothing;
```

Comment:
@garyb Do you mean simply removing this line, so that we return early in case of a positive result, but fall through otherwise?

Reply:
Yeah, exactly.